tools/regions.py

```diff
@@ -166,8 +166,11 @@ def merge_region_list(
         _, begin = merged.segments()[0]
         for start, stop in merged.segments()[1:]:
             pad_size = start - begin
-            merged.puts(begin, padding * pad_size)
-            begin = stop + 1
+            try:
+                merged.puts(begin, padding * pad_size)
+                begin = stop + 1
+            except: 				
```

Comment:
This exception clause is too wide-ranging. It catches everything, including system interrupts.
Also, this downgrades a fatal error into an info message that can easily be overlooked.
Also, there is little information in the message that allows the user to work out what happened (since the nature of the exception has been lost).

Reply:
agree.  I'll try to capture the specific error and report it.